app/assets/images/payment_icons/mobilepay.svg

```diff
@@ -1 +1 @@
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" width="38" height="24" viewBox="0 0 38 24" role="img" aria-labelledby="pi-mobilepay"><title id="pi-mobilepay">MobilePay</title><g fill-rule="evenodd" clip-rule="evenodd" id="pi-mobilepay-background_mobilepay"><path fill="#FFF" d="M36 24H2c-1.333 0-2-.7-2-2V2C0 .667.7 0 2 0h34c1.333 0 2 .7 2 2v20c0 1.333-.667 2-2 2z"/><path fill="none" stroke="#000" stroke-linecap="round" stroke-linejoin="round" stroke-miterlimit="3" stroke-opacity=".07" d="M36 24c1.333 0 2-.7 2-2V2c0-1.333-.7-2-2-2H2C.7 0 0 .7 0 2v20c0 1.333.7 2 2 2h34z"/></g><path fill-rule="evenodd" clip-rule="evenodd" fill="#5A78FF" d="M24.85 9.2c.601.133 1.134.35 1.601.65v4.8c-1.167-.634-2.434-.917-3.8-.851l1 2.3a1.28 1.28 0 010 1.101c-.167.3-.417.533-.75.7l-4.851 2a1.35 1.35 0 01-.5.1c-.3 0-.55-.066-.75-.2-.233-.133-.417-.35-.55-.649L11.65 7.9c-.133-.3-.133-.633 0-1s.383-.617.75-.75l4.85-2.05a1.303 1.303 0 011 0c.333.167.583.417.75.75l1.05 2.65c1.467-.567 2.75-.783 3.851-.65l.949 2.35zm-2.95 4.6c-1.334.033-2.467.316-3.4.851L16.55 9.9a.525.525 0 00.1-.2c.033-.034.067-.05.1-.05l.1-.1a.596.596 0 00.1-.15c.733-.7 1.55-1.25 2.45-1.65l-1.05-2.6c-.067-.1-.15-.2-.25-.3a.8.8 0 00-.35-.1c-.033 0-.117.017-.25.05l-4.85 2a.888.888 0 00-.35.35c-.033.2-.033.367 0 .5l4.65 11.25a.604.604 0 00.35.35.681.681 0 00.45-.05l4.9-2c.133-.033.25-.15.35-.351a.532.532 0 000-.449l-1.1-2.6z" id="pi-mobilepay-icon_mobilepay"/></svg>
```

Comment:
Same comments applied here: unused `id` attributes and border needs to follow other icons.

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="38" height="24" viewBox="0 0 38 24" role="img" aria-labelledby="pi-mobilepay"><title id="pi-mobilepay">MobilePay</title><g fill-rule="evenodd" clip-rule="evenodd"><path fill="#FFF" d="M36 24H2c-1.333 0-2-.7-2-2V2C0 .667.7 0 2 0h34c1.333 0 2 .7 2 2v20c0 1.333-.667 2-2 2z"/><path fill="none" stroke="#000" stroke-linecap="round" stroke-linejoin="round" stroke-miterlimit="3" stroke-opacity=".07" d="M36 24c1.333 0 2-.7 2-2V2c0-1.333-.7-2-2-2H2C.7 0 0 .7 0 2v20c0 1.333.7 2 2 2h34z"/></g><path fill-rule="evenodd" clip-rule="evenodd" fill="#5A78FF" d="M24.85 9.2c.601.133 1.134.35 1.601.65v4.8c-1.167-.634-2.434-.917-3.8-.851l1 2.3a1.28 1.28 0 010 1.101c-.167.3-.417.533-.75.7l-4.851 2a1.35 1.35 0 01-.5.1c-.3 0-.55-.066-.75-.2-.233-.133-.417-.35-.55-.649L11.65 7.9c-.133-.3-.133-.633 0-1s.383-.617.75-.75l4.85-2.05a1.303 1.303 0 011 0c.333.167.583.417.75.75l1.05 2.65c1.467-.567 2.75-.783 3.851-.65l.949 2.35zm-2.95 4.6c-1.334.033-2.467.316-3.4.851L16.55 9.9a.525.525 0 00.1-.2c.033-.034.067-.05.1-.05l.1-.1a.596.596 0 00.1-.15c.733-.7 1.55-1.25 2.45-1.65l-1.05-2.6c-.067-.1-.15-.2-.25-.3a.8.8 0 00-.35-.1c-.033 0-.117.017-.25.05l-4.85 2a.888.888 0 00-.35.35c-.033.2-.033.367 0 .5l4.65 11.25a.604.604 0 00.35.35.681.681 0 00.45-.05l4.9-2c.133-.033.25-.15.35-.351a.532.532 0 000-.449l-1.1-2.6z"/></svg>
```